### PR TITLE
Update urls-and-ip-address-ranges.md

### PR DESCRIPTION
**Pull request by msbemba**
Added to related topics-  if using 3rd party application, recommended downloading the Root CA and the Intermediate CA bundle on the application system. Referencing the certificate providers used by O365: https://docs.microsoft.com/en-us/microsoft-365/compliance/encryption-office-365-certificate-chains?view=o365-worldwide

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2759

File: microsoft-365/enterprise/urls-and-ip-address-ranges.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ Data columns shown are:
 [Managing Office 365 endpoints](managing-office-365-endpoints.md)
   
 [Troubleshooting Office 365 connectivity](https://support.office.com/article/d4088321-1c89-4b96-9c99-54c75cae2e6d.aspx)
+
+[Root CA and the Intermediate CA bundle on the 3rd party application system](https://docs.microsoft.com/microsoft-365/compliance/encryption-office-365-certificate-chains?view=o365-worldwide)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/urls-and-ip-address-ranges.md]:
```suggestion
[Root CA and the Intermediate CA bundle on the third-party application system](../compliance/encryption-office-365-certificate-chains.md)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**